linode_api4/objects/serializable.py

```diff
@@ -54,28 +57,57 @@ class JSONObject(metaclass=JSONFilterableMetaclass):
         )
     """
 
+    always_include: ClassVar[Set[str]] = {}
+    """
+    A set of keys corresponding to fields that should always be
+    included in the generated output regardless of whether their values
+    are None.
+    """
```

Comment:
I'm wondering if it'd make more sense to do something more similar to `omitempty` like in Go.